cmd/apiserver-watcher/run.go

```diff
@@ -267,3 +272,14 @@ func (sl *healthTracker) OnComplete(state *health.State) {
 		}
 	}
 }
+
+// filterAddrList filter a list of IP addresses by IP family
```

Comment:
filterAddrList filters a list of IP addresses and return list of addresses in IP family enabled by flag runOpts.ipv6

or some such thing that I'll be able to read and not forget the purpose of this whole thing ;)